outputs/graphite/graphite.go

```diff
@@ -0,0 +1,122 @@
+package graphite
+
+import (
+	"errors"
+	"fmt"
+	"github.com/influxdb/influxdb/client/v2"
+	"github.com/influxdb/telegraf/outputs"
+	"log"
+	"math/rand"
+	"net"
+	"strings"
+	"time"
+)
+
+type Graphite struct {
+	// URL is only for backwards compatability
+	Servers []string
+	Prefix  string
+	Timeout int
+	conns   []net.Conn
+}
+
+var sampleConfig = `
+  # TCP raw endpoint for your graphite instance.
+  servers = ["mygraphiteserver:2003"] # default "localhost:2003"
+  # Prefix metrics name
+  prefix = "" # default ""
+  # Timeout
```

Comment:
cool, can you add two things to the comment: 1) unit of the timeout 2) what is the timeout (ie, what is the thing that will timeout)